azerothcore/azerothcore-wotlk
# fix(DB/Creature): faction of ravenoak

Fix created by [Keira3](https://www.azerothcore.org/Keira3/)

## Changes Proposed:
-  change faction from 14 to 124, like the other NPCs near him (the faction 124 is the same on TC too)

## Issues Addressed:
- Closes #4226

## Tests Performed:
- tested in-game

## How to Test the Changes:
- `.go c id 5915`
- use horde and alliance character and check the faction, it should be enemy as horde and friendly as an alliance

## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
